dataactbroker/app.py

```diff
@@ -50,7 +46,7 @@ def createApp():
         cors = CORS(app, supports_credentials=False, origins=CONFIG_SERVICES['cross_origin_url'],
                     allow_headers = "*", expose_headers = "X-Session-Id")
     # Enable AWS Sessions
```

Comment:
Switched to "Enable DB session table handling" to cover both RDS and local cases